app/controllers/gobierto_participation/issues_controller.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module GobiertoParticipation
+  class IssuesController < GobiertoParticipation::ApplicationController
+    def index
+      @issues = current_site.issues.alphabetically_sorted
+    end
+
+    def show
+      @issue = find_issue
+      @issue_news = find_issue_news
+      @issue_news_updated = find_issue_news_updated
+      @issue_events = find_issue_events
+      @processes = current_site.processes.process.open
```

Comment:
Is there any reason you are not filtering the processes and the groups that belong to the current issue?